content/docs/command-reference/push.md

```diff
@@ -151,27 +151,20 @@ $ dvc push data.zip.dvc
 ## Example: With dependencies
 
 Demonstrating the `--with-deps` option requires a larger example. First, assume
-a [pipeline](/doc/command-reference/pipeline) has been setup with these
+a [pipeline](/doc/command-reference/dag) has been setup with these
 [stages](/doc/command-reference/run):
 
-```dvc
-$ dvc pipeline show
-data/Posts.xml.zip.dvc
-Posts.xml.dvc
-Posts.tsv.dvc
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
-```
+> model-posts  
+> post-features  
+> test-posts  
+> matrix-train
```

Comment:
Same

Reply:
Also, `model-posts` doesn't sound like an initial stage. Please use this list: `clean-posts`, `featurize`, `test-posts`, `matrix-train`